### ISSUE TITLE
How is callback supposed to set?

### ISSUE THREAD

**Issue by noxomix** [duarteroso/glfw#1]
Hello, I just want to know how the Callback is supposed to work, since I cannot use the C.GLFWwindow is private, so thought to use glfw.Window-struct instead but then its a type conversion, so how is it thought/supposed to set a Callback?

(And yes I managed to change some parts so the Library works in Vlang 4.0.1, just wondering if there is a way to archive yet, or you stopped further implementing)

Best Regards
Theo

**Comment by noxomix** [comment]
Solution:
I set C.GLFWwindow as public in the glfw.v , then I could set the callback. Thank you anyways :)

**Comment by duarteroso** [comment]
@noxomix When I did the wrapper function callbacks were not yet supported by V (0.2.x if I remember correctly). I am planning to give it a refresh closer to V 1.0 👍

**Comment by noxomix** [comment]
Ah ok i see. Yes its a Huge workload to keep libs updated while the Language Syntax changes / Strictness whatever.

Maybe I will create a Pull Request. I replaced your Semver by the vlib semver. 

Initially I just searched for a way to archive a rounded corner window, which requires the window element to be transparent , which i wasnt able to archive by Sokol, GG, Sfml or SDL2... And the working Glfw3 V-bindings arent implementing the Window Flags completly, so I just copied your Project as local Module. And did some changes to it, and maybe some arent Best Practice since im not a Professional at V and also absoloutly not in C :)

So yea not sure if a Pull Request Would destroy your Structure /naming scheme ect. 

I exported C.GLFWwindow to a Vlang pub Type  simply called GLFWwindow, and so i could do all the callback stuff now. And it works 👍
